website/content/docs/customization.md

```diff
@@ -54,13 +54,15 @@ body {
 
 ## `registerPreviewTemplate`
 
-Registers a template for a collection.
+Registers a template for a folder collection or an individual file in a file collection.
 
-`CMS.registerPreviewTemplate(collection, react_component);`
+`CMS.registerPreviewTemplate(name, react_component);`
 
 **Params:**
 
-* collection: The name of the collection which this preview component will be used for.
+* name: The name of the collection (or file for file collections) which this preview component will be used for.
+  * Folder collections: Use the name of the collection
+  * File collections: Use the name of the file
```

Comment:
Let's clarify what the "name" of the file is. I believe it is the slug (without file extension or path), correct?

Reply:
In the code I see it's `slug` for File Collections:
https://github.com/netlify/netlify-cms/blob/f801b192211b1e1953ced969a3093eab9e04e14d/packages/netlify-cms-core/src/reducers/collections.js#L94-L96

For Folder Collections it's the `name`:
https://github.com/netlify/netlify-cms/blob/f801b192211b1e1953ced969a3093eab9e04e14d/packages/netlify-cms-core/src/reducers/collections.js#L61-L63

But, in the admin config, it's not necessary to provide a `slug` for files:
https://github.com/netlify/netlify-cms/blob/f801b192211b1e1953ced969a3093eab9e04e14d/dev-test/config.yml#L55-L66

It seems like the `slug` ends up being the `name` for a file in a file collection, but I'm not sure where that is declared.

Reply:
I think I found where the `slug` is determined:
https://github.com/netlify/netlify-cms/blob/f801b192211b1e1953ced969a3093eab9e04e14d/packages/netlify-cms-core/src/reducers/collections.js#L78-L84

With that, do you think the current wording is okay? Or explicitly say the `name` field of the file in a file collection or folder collection.

Reply:
I think that will be fine, sorry!